widgets/common/src/deploymentsView/DeploymentsView.tsx

```diff
@@ -143,6 +148,10 @@ export const DeploymentsView: FunctionComponent<DeploymentsViewProps> = ({
         context.setValue('deploymentId', deployments[0].id);
     }
 
+    // eslint-disable-next-line react/destructuring-assignment
+    const mapOpen = (context.getValue(mapOpenContextKey) as boolean | undefined) ?? false;
```

Comment:
Wouldn't `!!` work here?

Reply:
It would :slightly_smiling_face: 

The line would be 

```suggestion
    const mapOpen = !!(context.getValue(mapOpenContextKey) as boolean | undefined);
```

I don't see a major difference and I don't mind so I'll use `!!~

Reply:
I just thought `as boolean | undefined` would be unnecessary in this case. Is it still needed?

Reply:
IMO it serves as additional documentation and tells you what values you can expect from the context :slightly_smiling_face: I don't think it hurts

Reply:
Since we are using `!` operator on the value I don't think this type documentation is of any use. Also, there is nothing that would ensure validity of this type - you could specify anything as the type here and it would still work. And if the actual type eventually changed this type specification would be misleading. I would still remove it, but I leave it up to you.

Reply:
Valid points :+1: I will merge it as-is to not ask for reviews again and prolong this PR, but I generally agree :slightly_smiling_face: